packages/neon/neon_news/lib/widgets/feed_icon.dart

```diff
@@ -1,33 +1,32 @@
 part of '../neon_news.dart';
 
 class NewsFeedIcon extends StatelessWidget {
-  const NewsFeedIcon({
+  NewsFeedIcon({
     required this.feed,
-    this.size = 48,
+    this.dimension = 48,
```

Comment:
I would keep this named size. The flutter Icon class also calls it size

Reply:
Bit it's called dimension when talking about a square.
https://api.flutter.dev/flutter/dart-ui/Size/Size.square.html